ova/generate_ova.sh

```diff
@@ -93,6 +98,7 @@ function main() {
   local WAZUH_VERSION=""
   local ELK_VERSION=""
   local STATUS=""
+  local cheksum="no"
```

Comment:
Typo => `cheksum`.

Reply:
@DFolchA

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/5a72389d9ddbedad13d02074112ad463a9d9e133